web/vue.config.js

```diff
@@ -1,6 +1,15 @@
 const MomentLocalesPlugin = require('moment-locales-webpack-plugin');
+const child_process = require('child_process');
 
-process.env.VUE_APP_VERSION = process.env.COMMIT_REF;
+function getVersion() {
+  try {
+    return child_process.execSync("git describe").toString();
```

Comment:
There's only one tag in the repo now (`0.1.0`) and it's annotated. If we anticipate ever adding additional [non-annotated tags](https://stackoverflow.com/a/11514139), then this should use `git describe --tags`. If all future tags will be annotated, then this is fine as-is.